### ISSUE TITLE
Running scyllatop on AMI 0.19 fail

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#1037]
```
$ scyllatop 
ERROR: could not connect to /var/run/collectd-unixsock. [Errno 2] No such file or directory
```

**Comment by tzach** [comment]
Look like collectd is not configure and not running by default on AMI.

**Comment by dorlaor** [comment]
It does work on HEAD, the only issue I had is that it pollutes the log with collectd msg

**Comment by haarcuba** [comment]
scyllatop.py --print-config will suggest this for your collectd configuration:

LoadPlugin unixsock

<Plugin unixsock>
   SocketFile "/var/run/collectd-unixsock"
   SocketGroup "wheel"
   SocketPerms "0660"
   DeleteSocket false
</Plugin>


also make sure you have permissions to connect to that socket.

**Comment by haarcuba** [comment]
turns out the Network plugin is also not enabled, and collectd is not running by default.

IMO for the next release we should have collectd configured properly by default.

**Comment by penberg** [comment]
@slivne @dorlaor This is fixed in ``master``. Do we want to backport the fix to 1.0?

**Comment by avikivity** [comment]
Yes.
